wakatime.plugin.zsh

```diff
@@ -74,6 +95,8 @@ _wakatime_heartbeat() {
     &>/dev/null </dev/null &!
 }
 
+check_file
```

Comment:
Let's add an option to enable this behavior. I think that having it off by default is a good thing.

Reply:
Alright, 

How Would You Suggest We Let The Users Know About The Availability Of The Option?

Perhaps A Prompt Such As `would you like to enter your api key? [y/N]` Or Something Like That.

Reply:
Just a note in the docs is fine. Pushing some prompts in the shell is not very friendly.

Reply:
Alright, That Makes Sense.

Thank You Very Much Sir.

Would Detecting The Presence Of An Environmental Variable To Trigger The Behaviour Be Okay?

Reply:
Thank You Very Much For Taking Time To Review My Contributions  Again.